### ISSUE TITLE
When using background image in MainLandingPage crashes unpackaged app

### ISSUE THREAD

**Issue by DFFspace** [ghost1372/DevWinUI#14]
I am using the DevWinUI preview 3 and I noticed my unpackaged app crashes when using a header image inside the MainLandingPage. Taking the header image out of there fixes the issue and doesn't crash it.

**Comment by ghost1372** [comment]
Can you upload a repro/sample project?

**Comment by ghost1372** [comment]
I don't see a problem.🫤
![Animation](https://github.com/user-attachments/assets/f71d547e-b381-40eb-837b-8e69b460bed0)

**Comment by DFFspace** [comment]
I am checking what the issue could be on my end, as I was creating a fresh new app to reproduce it, it seems to even just crash with and without HomeLandingPage and just WASDK 1.7 Exp1 and DevWinUI: 

![image](https://github.com/user-attachments/assets/a3026a34-6ab4-4bc4-b13c-9a38aedef77f)

Now when I go back to my main app it breaking now as well, while before it just worked, so maybe there is something else going on.....

**Comment by DFFspace** [comment]
But if you want to check it out I uploaded the repo of it

https://github.com/DFFspace/HomeLandingPageCrash

**Comment by ghost1372** [comment]
> I am checking what the issue could be on my end, as I was creating a fresh new app to reproduce it, it seems to even just crash with and without HomeLandingPage and just WASDK 1.7 Exp1 and DevWinUI:
> 
> ![image](https://private-user-images.githubusercontent.com/69241933/395111443-a3026a34-6ab4-4bc4-b13c-9a38aedef77f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5OTkxOTAsIm5iZiI6MTczMzk5ODg5MCwicGF0aCI6Ii82OTI0MTkzMy8zOTUxMTE0NDMtYTMwMjZhMzQtNmFiNC00YmM0LWIxM2MtOWEzOGFlZGVmNzdmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEyVDEwMjEzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUzMTU0Y2MwZjMxZTIxY2ZmZThjYjA1YzQ5N2QyZDk3NzM3MDJiZjQ2NmM4OGE2ZGNlYjFkMmM0MTk1M2UxZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.NLtBLkNhPE3qU-SSyd6ZS1any8CoMijqe06hmC6uO5U)
> 
> Now when I go back to my main app it breaking now as well, while before it just worked, so maybe there is something else going on.....

Have you installed v1.7-epx1 runtimes? you need it for unpackaged mode.
https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/experimental-channel#version-17-experimental-170-experimental1

also you can find more info here:

![image](https://github.com/user-attachments/assets/26512dfb-2fba-4d18-9134-98f4d2532ead)

**Comment by ghost1372** [comment]
> But if you want to check it out I uploaded the repro of it
> 
> https://github.com/DFFspace/HomeLandingPageCrash

your issue is:
![image](https://github.com/user-attachments/assets/1aafb277-fbe3-4dbc-8423-18fe7539519b)

**Comment by ghost1372** [comment]
> But if you want to check it out I uploaded the repro of it
> 
> https://github.com/DFFspace/HomeLandingPageCrash

you need to change Build Action to Page as you can see it changed to None.
![image](https://github.com/user-attachments/assets/9f19ce13-e193-485f-9ac4-5af109a46886)

**Comment by DFFspace** [comment]
Ah I see, thanks for pointing that out. I the ThemeResources method as I thought I had to use something like this and forgot to actually copy it, thinking it would do this all automatically.

I took ThemeResources out to where how I use it on my main app by just using:

`HeaderImage="ms-appx:///Assets/HomePage.png"
HeaderImageHeight="750"`

Now I can reproduce the issue when using the above lines it fails to load, taking these lines out opens the app fine.

![image](https://github.com/user-attachments/assets/ff466d11-ff87-4e09-b60e-09e09c634ddc)

Adding the two lines back results this message:

![image](https://github.com/user-attachments/assets/143d34f0-fcd1-415b-bb87-13ee4101ffdb)
